buddyboss/buddyboss-platform
# Added Feature: Show relevant Feed - at the global news feed page #1311

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1311 .

### How to test the changes in this Pull Request:

1. GO to BuddyBoss settings > Activity
2. Check **Show Relevant Feed**
3. Go to the global new feed page, and see, all posts relevant to members.

### Proof Screenshots or Video

https://www.loom.com/share/f32055a9a3fe4c459b242110b3e1e553





### Changelog entry

> Added Feature: Show relevant Feed - at the global news feed page
